PowerEditor/installer/functionList/php.xml

```diff
@@ -176,7 +176,7 @@
 							|   o(?:r|bject)
 							|   p(?:r(?:i(?:nt|vate)|otected)|ublic)
 							|   re(?:quire(?:_once)?|turn)
-							|   s(?:t(?:atic|ring)|witch)
+							|   s(?:elf|t(?:atic|ring)|witch)
```

Comment:
There's no proof why self is necessary for the functionlist, without added unit test:
> The PR for the enhancement of Function List parser should also include unit test. Please refer [here](https://npp-user-manual.org/docs/function-list/#contribute-your-new-or-enhanced-parser-rule-to-the-notepad-codebase) for more information.

Ref: https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/CONTRIBUTING.md#guidelines-for-pull-requests

Reply:
I have tried to understand how to add a unit test for this, but having taken a look I have no idea.

Since I don't know enough to be certain whether the FunctionList.xml changes are necessary, and since if they are I have no idea how to add a unit test, I am going to revert the FunctionList changes and stick with just the keyword.